src/app/editor/components/node-editor/node-editor.component.ts

```diff
@@ -31,6 +32,8 @@ export class NodeEditorComponent implements OnInit, OnDestroy {
     nodePathRouterLink: any[];
     nodePath: string;
     nodeTitle = '';
+    //isSaving$: Observable<boolean>;
```

Comment:
Add a todo detailing (in brief) the solution with the modal then go with this for now.